doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -3,164 +3,239 @@
 
 Workaround: None.
-endif::DM[]
 
-ifdef::PAM[]
-== OpenShift
+.Nodes and distribution lines do not fully align horizontally [https://issues.jboss.org/browse/JBPM-8555[JBPM-8555]]
+
+Issue: If you drag a node, the align and distribution mechanism correctly displays the alignment lines and also applies the snap-to-line behavior when the mouse moves. However, it is not possible to align the nodes horizontally which means that it is difficult to have the connectors precisely aligned horizontally.
+
 
-.User password is displayed in the Readiness and Liveness probes [https://issues.jboss.org/browse/RHPAM-1829[RHPAM-1829]]
-Issue: If you deploy {PRODUCT} on OpenShift, the user password is displayed in both the Readiness and Liveness probes.
+Steps to reproduce:
+
+. Create a process.
+. Try to align a node and connector horizontally.
+
+Expected result: The node and connector are aligned.
+
+Actual result: The node and connector are not aligned.
 
 Workaround: None.
-endif::PAM[]
 
-== {PLANNER}
+.The multiple instance properties of a multiple instance sub-process are not visible for nodes inside of a multiple instance sub-process [https://issues.jboss.org/browse/JBPM-8512[JBPM-8512]]
+
+Issue: If you model a process with a multiple instance sub-process as a multiple instance node, the multiple instance Data Input and multiple instance Data Output variables are not visible for nodes inside of the multiple instance sub-process.
+
+
+Steps to Reproduce:
+
+. Create a multiple instance sub-process for execution by selecting the multiple instance collection input/output from an existing process level list and setting new variables for multiple instance data input and output.
+. Add a user task inside of the multiple instance sub-process.
+. Try to configure  a user task so that is uses multiple instance data input/output.
 
-.`ScoreCalculationCountLimit` termination does not work with multi-threaded solving [https://issues.jboss.org/browse/RHDM-740[RHDM-740]]
+Actual Result: The new variables for multiple instance data input and output are not listed in the data I/O editor Source/Target fields.
 
-Issue: {PLANNER} configured with the score calculation count termination (`ScoreCalculationCountLimit`) does not terminate. The score calculation count termination is primarily used for testing.
+Expected Result: The new variables for multiple instance data input and output are listed in the data I/O editor Source/Target fields and you can use the new variables for multiple instance data input and output.
 
-Workaround: Use another type of termination.
+Workaround: To use variables for multiple instance data input and output, define them in *Process Data* -> *Process Variables* before creating the multiple instance sub-process for execution.
 
-//ifdef::PAM[]
-//== Smart Router
+.Data output associations do not work correctly when they are declared twice [https://issues.jboss.org/browse/JBPM-8489-[JBPM-8489]]
```

Comment:
Hey @emmurphy1 
Just added the missing information about this issue, see the updated [JBPM-8489](https://issues.jboss.org/browse/JBPM-8489)
Thanks!

Reply:
Updated, please review.

Reply:
it looks good to me, thx :+1: